calculator/parser.py

```diff
@@ -139,3 +182,96 @@ def rpn(tokens):
                 ",".join(str(t) for t in tokens)
             )
         )
+
+
+def infix(token_list):
+
+    """
+    Parses Infix notation and returns the equivilant RPN form
+
+    Parameters
+    ==========
+    token_list : list
+        The list of infix tokens
+
+    Returns 
+    --------
+    output : list
+        This is the output queue where the final answer is arranged
+    
+    """
+
+    # Output Queue
+    output = []
+
+    # Operators Stack
+    operators = []
+
+    iteration_num = 1
```

Comment:
Name this better/different, please.

Reply:
In particular, please specify what it is counting.  This name would imply that it is the index of the loop through the `token_list`, whereas I think what this actually is is the number of tokens you've *fully* processed (i.e. a token can be deferred until you hit an end-paren or somesuch).  I would like a better name and a comment indicating what it's actually doing.

Reply:
👍